### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
Fixes all the tests 

# Type of Change:
- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: vms/shift/tests/test_viewVolunteerShift.py [hunk 1]
```diff
@@ -171,28 +171,6 @@ def test_view_with_assigned_and_unlogged_shift(self):
         self.assertEqual(upcoming_shift_page.get_shift_start_time(), '9 a.m.')
         self.assertEqual(upcoming_shift_page.get_shift_end_time(), '3 p.m.')
 
-    def test_log_hours_and_logged_shift_does_not_appear_in_upcoming_shifts(self):
```
Comment (Monal5031) [inline vms/shift/tests/test_viewVolunteerShift.py]: The later part of the test is still valid right?  `logged hours not appearing in upcoming shifts`

**Comment by abhishekarya286** [comment]
Hi @nida @kamsuri. Please review.

File: vms/shift/tests/test_viewVolunteerShift.py (hunk 1 above)
Comment (anjali-dhanuka) [reply vms/shift/tests/test_viewVolunteerShift.py]: But @Monal5031  I can't log hours for a future shift, so this function doesn't make sense.

Comment (anjali-dhanuka) [reply vms/shift/tests/test_viewVolunteerShift.py]: @abhishekarya286 @Monal5031 and how can I log hours for a future shift?

Comment (abhishekarya286) [reply vms/shift/tests/test_viewVolunteerShift.py]: He was just talking about the code after line 188. It implies if you don't sign up for a shift, there should be nothing on the upcoming shifts page.

Comment (anjali-dhanuka) [reply vms/shift/tests/test_viewVolunteerShift.py]: But this test already exists

Comment (anjali-dhanuka) [reply vms/shift/tests/test_viewVolunteerShift.py]: https://github.com/anjali-dhanuka/vms/blob/531853b4c5f8fb5235b819582dab8697b6ad725d/vms/shift/tests/test_viewVolunteerShift.py#L151

Comment (abhishekarya286) [reply vms/shift/tests/test_viewVolunteerShift.py]: Yes, then it's fine. I was not aware about this test.

**Pull request merged**